lib/utils/npm-usage.js

```diff
@@ -19,7 +19,7 @@ npm help <term>    search for help on <term>${usesBrowser}
 npm help npm       more involved overview${usesBrowser}
 
 All commands:
-${allCommands(npm)}
+${await allCommands(npm)}
```

Comment:
The above comment probably isn't clear because I can't comment on unchanged lines, but I think we should explicitly mark `allCommands` as an `async` function.

Reply:
will do